randovania/games/dread/json_data/Elun.txt

```diff
@@ -176,6 +176,8 @@ Extra - asset_id: collision_camera_003
   * Pickup 114; Major Location? False
   * Extra - actor_name: item_energytank_000
   * Extra - actor_def: actordef:actors/items/item_energytank/charclasses/item_energytank.bmsad
+  > Door to Chozo Soldier Arena
+      Lay Bomb
```

Comment:
So, why add this connection back after removing it before after i said to remove it?